crates/uv/tests/run.rs

```diff
@@ -89,6 +91,7 @@ fn run_with_python_version() -> Result<()> {
     // This time, we target Python 3.11 instead.
     let mut command = context.run();
     let command_with_args = command
+        .arg("--show-environment")
```

Comment:
I considered just setting this as a default in `command.run()` for the test context, what do you think?

Reply:
That'd be okay with me, I'd just document it.

Reply:
One problem is that you can't drop the flag.

Reply:
Yeah that's why I "un-did" that approach. I could instead set is an env var and then you can `env_remove` like with `--exclude-newer`.